### DISCUSSION TITLE
!important in :style="{}"

### DISCUSSION THREAD

**Discussion by RobinManoli** [alpinejs/alpine#4473] [1. Help]
It seems not to work to use !important in this kind of code:
:style="{color:'red !important'}"

Is there any way to make it work?

Should be a good feature to make it work otherwise

**Comment by SimoTod** [comment]
I don't think that syntax is supported (it only works with classes).

https://alpinejs.dev/directives/bind

**Comment by ekwoka** [comment]
This is limited by how the CSS api works on elements.

https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration/setProperty

So the object is basically mapped to the `key` and `value` here.

I could see a way to support it by having keys that start with `!` then pass in a `priority`...

**Comment by RobinManoli** [comment]
Well, a workaround to use a variable with an !important style is to use :style as a string
red = 'color:red !important;'
:style="red"
